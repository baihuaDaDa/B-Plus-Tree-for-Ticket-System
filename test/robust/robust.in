75
insert a 10
insert a 20
insert a 30
insert a 40
insert a 50
insert a 60
insert a 70
insert a 80
insert a 90
insert a 100
insert a 110
insert a 120
insert a 130
insert a 140
insert a 43
insert a 47
insert a 83
insert a 87
insert a 103
insert a 107
insert a 123
insert a 127
insert a 133
insert a 137
find a
delete a 10
delete a 20
delete a 30
delete a 40
delete a 50
delete a 60
delete a 70
delete a 80
delete a 90
delete a 100
delete a 110
delete a 120
delete a 130
delete a 140
delete a 43
delete a 47
delete a 83
delete a 87
delete a 103
delete a 107
delete a 123
delete a 127
delete a 133
delete a 137
find a
insert a 10
insert a 20
insert a 30
insert a 40
insert a 50
insert a 60
insert a 70
insert a 80
insert a 90
insert a 100
insert a 110
insert a 120
insert a 130
insert a 140
insert a 43
insert a 47
insert a 83
insert a 87
insert a 103
insert a 107
insert a 123
insert a 127
insert a 133
insert a 137
find a