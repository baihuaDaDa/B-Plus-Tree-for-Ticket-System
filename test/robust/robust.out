10 20 30 40 43 47 50 60 70 80 83 87 90 100 103 107 110 120 123 127 130 133 137 140 
null
10 20 30 40 43 47 50 60 70 80 83 87 90 100 103 107 110 120 123 127 130 133 137 140 
